nix/haskell.nix

```diff
@@ -100,16 +100,26 @@ let
 
         # cardano-node will want to write logs to a subdirectory of the working directory.
         # We don't `cd $src` because of that.
-				#
-				# TODO: Using the configuration dir works for mainnet, but to add testnet
-        # support, we need to retrieve it properly.
-        packages.cardano-wallet-byron.components.benchmarks.restore =
+        packages.cardano-wallet-byron.components.benchmarks.testnet-restore =
```

Comment:
Some nix machinery relies on this being named `benchmarks.restore` and not `benchmarks.testnet.restore` or `benchmarks.testnet-restore`.

This fails with
```
Setup: Unknown build target 'bench:testnet-restore'.
There is no benchmark component 'testnet-restore'.
```

Not sure what to do. @rvl?

Reply:
I could pass in `NODE_{MAINNET, TESTNET}_{CONFIG, GENESIS}` and let the restore benchmark decide which config to use based on `--mainnet` or `--testnet` args – such that only _one_ nix wrapper is needed.

Not super-elegant, but should work.